### PR TITLE
more doc fixes

### PR DESCRIPTION
**Pull request by fjy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ More information about Druid can be found on <http://www.druid.io>.
 
 ### Documentation
 
-You can find the [latest Druid Documentation](http://druid.io/docs/latest/) on
+You can find the [documentation for the latest Druid stable](http://druid.io/docs/latest/) on
```
Comment (gianm) [inline README.md]: 'release' instead of 'stable' sounds better imo

File: docs/content/operations/other-hadoop.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ layout: doc_page
 ---
 # Work with different versions of Hadoop
 
-## Include Hadoop dependencies
+## Including Hadoop dependencies
```
Comment (gianm) [inline docs/content/operations/other-hadoop.md]: change 'work' to 'working'? the tenses no longer agree

File: docs/content/querying/dimensionspecs.md [hunk 3]
```diff
@@ -396,6 +396,10 @@ or without setting "locale" (in this case, the current value of the default loca
 
 ### Lookup DimensionSpecs
 
+<div class="note caution">
+Lookups are an experimental feature.
```
Comment (gianm) [inline docs/content/querying/dimensionspecs.md]: some details about what this means would be nice.

do either of these apply?

- api may change in backwards incompatible ways
- 'beta' - not yet validated across a variety of production clusters

Comment (gianm) [inline docs/content/querying/dimensionspecs.md]: or link to the 'experimental features' page if that already explains what experimental means.

**Comment by gianm** [comment]
:+1:

**Comment by drcrallen** [comment]
Conflicts with 0.9.0 branch. @fjy needs fixes to backport

**Pull request merged**